Packs/SymantecDLP/Integrations/SymantecDLPV2/SymantecDLPV2.py

```diff
@@ -0,0 +1,909 @@
+    if remediation_status_name:
+        data.update({'preventOrProtectStatus': remediation_status_name})
+    if remediation_location:
+        data.update({'remediationLocation': remediation_location})
+    if severity:
+        data.update({'severity': severity})
+    custom_attributes_list = build_custom_attributes_update(custom_attributes)  # type: ignore
+    if custom_attributes_list:
+        data.update({'incidentCustomAttributes': custom_attributes_list})
+    return data
+
+
+def build_custom_attributes_update(custom_attributes: List[str]):
+    """
+    Builds the custom_attributes_list that the user wants to update.
+    :param custom_attributes: The custom attributes the user wants to update
+    :return: A list of custom attributes
+    """
+    custom_attributes_list = []
+    for attribute in custom_attributes:
+        splitted_att = attribute.split(':')
+        if len(splitted_att) != 2:
+            raise DemistoException('Error: custom_attributes argument format is {columnIndex}:{newValue}. E.g: 1:test')
+        attribute_index = splitted_att[0]
+        if not attribute_index.isdigit():
+            raise DemistoException('Error: The attribute index must be an integer.')
+        attribute_value = splitted_att[1]
+        custom_attributes_list.append({"columnIndex": int(attribute_index), "value": attribute_value})
+    return custom_attributes_list
+
+
+def get_incident_details_fetch(client, incident):
+    """
+    Create incident details dict for each incident pulled from the fetch
+    """
+    incident_details = {}
+    try:
+        incident_id = incident.get('incidentId')
+        incident_type = incident.get('messageSource', '')
+        static_attributes = client.get_incident_static_attributes_request(incident_id)
+        editable_attributes = client.get_incident_editable_attributes_request(incident_id)
+        incident_details = get_common_incident_details(static_attributes, editable_attributes,
+                                                       args={"custom_attributes": "all"}, isfetch=True,
+                                                       incident_type=incident_type)
+    # In case of getting 401 (Unauthorized incident) - will get missing data
+    except DemistoException as e:
+        if '401' in str(e):
+            incident_details = get_details_unauthorized_incident(incident)
+    return incident_details
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+    """
+    Performs basic get request to get item samples
+    """
```

Comment:
1. remove the docstring please
2. Let's make sure we also check the validity of params in the test module